src/main/java/com/linkedin/replica/serachEngine/databaseHandlers/impl/SqlHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.linkedin.replica.serachEngine.databaseHandlers.impl;
+
+import java.util.List;
+
+import com.linkedin.replica.serachEngine.databaseHandlers.DatabaseHandler;
+import com.linkedin.replica.serachEngine.models.Company;
+import com.linkedin.replica.serachEngine.models.Job;
+import com.linkedin.replica.serachEngine.models.User;
+import com.linkedin.replica.serachEngine.models.Post;
+
+/**
+ * Implementation of SqlHandler which is responsible for serving specific request from SQL database
+ */
+public class SqlHandler implements DatabaseHandler {
```

Comment:
Do we need this file? I think SQL is not concerned with any search functionality

Reply:
I kept it as to be as a reference for everyone else